text/0000-2020-roadmap.md

```diff
@@ -0,0 +1,122 @@
+
+- **Documentation.** Promote documentation to a first class citizen in Grin.
+- **Exchange integrations.** Provide better integration instructions for exchanges. Best practices, examples, and components as required.
+- **Wallet integrations.** Provide better guidance for wallet projects on how to support Grin.
+- **Merchant integrations.** Provide instructions for merchants on how to accept payments in Grin from their users in a manner that is privacy preserving and secure.
+
+## Community
+
+- **Onboarding.** Make it easier for newcomers to begin contributing to the project.
+- **grincon2.** Better planning, better lead times, raise quality, and attendance.
+- **Website.** Improve communication and content on the Grin website.
+- **Meetups.** Facilitate meetup organization and "local community start packages".
+
+## Governance
+
+- **Moderation.** Establish an indepent Moderation team.
+- **Vision.** Define Grin trade-offs and design goals.
+- **Funding proposals.** Formalize a process, encourage submissions.
+- **Resilience.** Avoid creating any more single point of failures, share access and privileges.
+- **2021 Roadmap.** Improve on the roadmap planning process for the next year. More structure, milestones, and make it more inclusive.
+
+# Drawbacks
+
+[drawbacks]: #drawbacks
+
+Having a defined roadmap might give the false impression to contributors that contributions in other areas are not welcomed. It might also introduce more hierarchy, politics, and bureaucracy into Grin's processes.
+
+# Rationale and alternatives
+
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+As outlined, the focus is on improving what already exists today rather than introducing new functionality and features. This is deliberate. As the project is already spread out thin in terms of resources and contributors, care should be taken not to increase the surface area further.
+
+# Prior art
+
+[prior-art]: #prior-art
+
+This RFC been heavily inspired by Rust's 2019 roadmap process.[1]
+
+In October 2019, there was a community call for ideas posted to the Grin Forum.[2] Below are excerpts from some of the submissions (by no means exhaustive):
+
+*AlexGSG:*
+> there is a problem to withdraw from exchanges via HTTPS, it is drives the price down , because much easy to sell , than to buy!
+
+*Kurt:*
+> Continue looking at techniques helping obfuscate tx graph (dandelion iterations, zero-knowledge cryptography to unlink TXO,…)
+
+*Phyro:*
+> If a full node can scale on a mobile phone for many years to come start work on that, otherwise, research light client options
+
+*Andro:*
+> The one thing that I want to see in 2020 is user friendly transactions and wallets.
+
+*notguilty:*
+> Features are fun and nice but secondary to basic reliability. I would like to see a solid foundation...
+
+*Roelsmajor*
+> When [FlyClient paper] https://eprint.iacr.org/2019/226?
+
+
+# Unresolved questions
+
+[unresolved-questions]: #unresolved-questions
```

Comment:
We should probably make an effort to resolve some of these before merging this. Doesn't seem very useful without answering the 2nd one, for example.